tests/run_opcuabridge_ostree_repo_sync_test.sh

```diff
@@ -25,11 +25,9 @@ fi
 sleep 3
 trap 'kill %1' EXIT
 
-if [ "$3" == "valgrind" ]; then
-  $BIN_DIR/run-valgrind $BIN_DIR/opcuabridge-ostree-repo-sync-client $TEST_SRC_REPO $PORT
-else
-  $BIN_DIR/opcuabridge-ostree-repo-sync-client $TEST_SRC_REPO $PORT
-fi
+# $3 can be set to 'valgrind' (or the run-valgrind wrapper)
```

Comment:
Should we also modify line 19 to look like this? It hard-codes arguments to valgrind.

Reply:
That makes sense to me.